build.gradle

```diff
@@ -17,8 +17,16 @@ repositories {
 	}
 }
 
+// Comment out this block if you're using Chocoloom
+// Comment when using loom 0.10
```

Comment:
could this be rewritten as "Comment out this block if you're using Chocoloom or loom 0.10?